.php_cs

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once './vendor/autoload.php';
+require __DIR__.'/vendor/sllh/php-cs-fixer-styleci-bridge/autoload.php';
```

Comment:
@Soullivaneuh maybe the conflict wasn't longer here but the aim of this autoloader is not to fix conflicts.